forgettable-frontend/src/pages/SettingsPage/SettingsPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import NavBar from './components/NavBar/NavBar';
+import classes from './SettingsPage.module.css';
+import {getAuth, signOut} from 'firebase/auth';
+import DarkMode from './DarkMode';
+
+function SettingsPage() {
+  const signOutHandler = ()=>{
+    const auth = getAuth();
+    signOut(auth).then(() => {
+      // Sign-out successful.
+    }).catch((error) => {
+      // An error happened.
+    });
+  };
+
+  return (
+    <div className='SettingsPage'>
+      <NavBar></NavBar>
+      <div className={classes.title}>Settings:</div>
+      <div className={classes.account}>Account:</div>
+      <div className={classes.border}></div>
+      <div className={classes.name}>Name</div>
+      <div className={classes.content}>user name</div>
+      <div className={classes.email}>Email</div>
+      <div className={classes.content}>username@gmail.com</div>
```

Comment:
We should probably retrieve user information from backend for these two fields?

Reply:
Currently, the backend doesn't store email. A get/users would only return the users name, which is currently also accessible in localstorage. If _really_ wanted email for MVP we could make a call to firebase to get the full user info, or store the email in localstorage the same way we do the display_name, but I think it would be better to push this to next version and make tickets for it so it can be done properly. 

Likely, the ideal solution here is that when we call get/users, the backend makes a call to firebase and returns user information directly from the source. So we can make a ticket for that for future work.

In terms of this bit, I say we just display the username from local storage (as is done on the front page) but take out the email field for now. @rainasong  @jyao244  thoughts?

Reply:
Only displaying username sounds good for now!

Reply:
it can display user name now

Reply:
email field have been taken out